trunk/NDHMS/Routing/Reservoirs/Persistence_Level_Pool_Hybrid/module_persistence_levelpool_hybrid.F

```diff
@@ -0,0 +1,350 @@
+! This module defines and instantiates objects
+! for a hybrid persistence levelpool type
+! reservoir. The hybrid reservoir struct
+! inherits input and output stucts from the
+! reservoir base module and calls instantiation
+! of these into sub-objects. The hybrid
+! reservoir struct also points to structs for
+! hybrid parameters and state and calls
+! instantiation of these into sub-objects.
+! A pointer to a levelpool reservoir object
+! is also held in state, and this module
+! instantiates that levelpool object. There
+! is also a subroutine to run hybrid release
+! that is derived from the reservoir base
+! struct interface to run release.
+
+! FIXME: Change value for WCOS
+#define log_warning 6
+
+module module_persistence_levelpool_hybrid
+
+    use module_persistence_levelpool_hybrid_parameters
+    use module_persistence_levelpool_hybrid_state
+    use module_levelpool
+    use module_reservoir_utilities
+    use module_reservoir_base
+    use module_read_timeslice_data
```

Comment:
again, `only:`s would make this much clearer to read.

Reply:
THis is basically a pep8 thing as well. See the last bullet of 
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
@jdmattern-noaa fixing